AWSAuthSDK/Sources/AWSMobileClient/Internal/AWSCognitoAuth+Extensions.m

```diff
@@ -57,7 +58,41 @@ - (instancetype)initWithAppClientId:(NSString *) appClientId
           signOutUriQueryParameters:(nullable NSDictionary<NSString *, NSString *> *) signOutUriQueryParameters
             tokenUriQueryParameters:(nullable NSDictionary<NSString *, NSString *> *) tokenUriQueryParameters
        userPoolServiceConfiguration:(nullable AWSServiceConfiguration *)serviceConfiguration {
+    return [self initWithAppClientId:appClientId
```

Comment:
As above, do we need both the legacy and new methods for this internal API?

Reply:
Removed the old api.